airbyte-integrations/connectors/source-mailchimp/source_mailchimp/streams.py

```diff
@@ -138,6 +138,43 @@ def path(self, **kwargs) -> str:
         return "lists"
 
 
+class ListMembers(IncrementalMailChimpStream):
+    cursor_field = "last_changed"
+    data_field = "members"
+
+    def read_records(self, sync_mode: SyncMode, cursor_field: List[str] = None, stream_slice: Mapping[str, Any] = None, stream_state: Mapping[str, Any] = None) -> Iterable[Mapping[str, Any]]:
+        lists_stream = Lists(**kwargs)        
```

Comment:
Try explicitly passing in the authenticator: i.e lists_stream = Lists(authenticator=self.authenticator)

Reply:
This works for the stream :D